### ISSUE TITLE
Latest releases breaks spaces in PRs

### ISSUE THREAD

**Issue by evgenyrodionov** [sindresorhus/refined-github#590]
For example: https://github.com/rob3n/yard-frontend/pull/4/files
https://github.com/rob3n/yard-frontend/blob/770ad32d0b60f42aff3c7e61edb5fbfdaecb5bc1/public/index.html

![image](https://user-images.githubusercontent.com/775692/27977729-b95642a6-6374-11e7-95c5-e5dd76583400.png)

But file have spaces
![image](https://user-images.githubusercontent.com/775692/27977754-d79b63f4-6374-11e7-97f7-5233c33d8757.png)

**Comment by arkon** [comment]
I'm seeing this too. I think it's only for diffs that are loaded later.

**Comment by bfred-it** [comment]
Can't reproduce:

<img width="402" alt="screen shot 2017-07-08 at 16 43 57" src="https://user-images.githubusercontent.com/1402241/27983967-d452c182-63fc-11e7-8682-ecba1c7cec51.png">

> I'm seeing this too. I think it's only for diffs that are loaded later.

What do you mean? What are the steps?

**Comment by bfred-it** [comment]
Located issue: it's `removeDiffSigns`

**Comment by bfred-it** [comment]
Uhh the whitespace is collapsing once it's no longer prefixed by a `-` or `+`. The cause seems to be `pre-wrap` but I'm not sure if we should disable that.

![pre-wrap](https://user-images.githubusercontent.com/1402241/27984052-1371c808-63fe-11e7-8a86-8471cf8eb714.gif)

**Comment by bebehei** [comment]
Hi, I installed today refined-github the first time. I also experienced a related bug with whitespacing. This bug here is not the same, but may get fixed in the same time.

My commit adds a single line without any preceeding whitespace, but it get's displayed with a small extra indentation (which is confusing).


Correct: (plain github)
![screenshot from 2017-07-09 18-45-49](https://user-images.githubusercontent.com/2073458/27995940-6aef495a-64d7-11e7-846c-3cd88d64d6bf.png)

Wrong: (refined github enabled)
![screenshot from 2017-07-09 18-45-15](https://user-images.githubusercontent.com/2073458/27995941-6db3f398-64d7-11e7-93d8-1755fc8315cc.png)

The Raw commit is here: dunst-project/dunst@717c747a8caf0f823da0e4e45bbb632ef46c68ee
